### ISSUE TITLE
Upload summary

### ISSUE THREAD

**Issue by reedy** [yarl/vicuna#16]
It'd be nice if an upload summary could be added, rather than just "VicuñaUploader X.XX"

**Comment by michal-josef-spacek** [comment]
@reedy ok, that's nice idea. Do you have a proposal for text in summary? Or some extra field?

**Comment by reedy** [comment]
It's hard to know what I was thinking about 9 years ago...

Is the upload summary still just "VicuñaUploader X.XX"?

**Comment by michal-josef-spacek** [comment]
@reedy I understand.. So issue is about writing some field for "upload summary".

**Comment by CodingLama** [comment]
> It's hard to know what I was thinking about 9 years ago...
> 
> Is the upload summary still just "VicuñaUploader X.XX"?

Yes, it is just that. It might be a question to @yarl whether there is a need to have VicuñaUploader X.XX in the upload summary, or whether you can just add what you want in it. At least VicuñaUploader X.XX is not a tag in recent changes yet, so it could be choosable from a field somewhere e.g. in settings:general

**Comment by reedy** [comment]
A minimum improvement would be something like "Uploaded file with VicuñaUploader X.XX"

**Comment by CodingLama** [comment]
> A minimum improvement would be something like "Uploaded file with VicuñaUploader X.XX"
Yes, we can apply that as a default edit summary.
